.asf.yaml

```diff
@@ -49,7 +49,6 @@ github:
     - jun-he
     - marton-bod
     - samarthjain
-    - findepi
```

Comment:
@findepi: Congratulations on being committer :) 

I would also like to remove the people who are inactive from 6 months or more in iceberg repo from this list to make space for new people who are active and intersted in project management. What do you guys think?

Reply:
And @jbonofre: Since you are more active on the release side, I think we should add you here :)

Reply:
@ajantha-bhat 😄 Do you want to add me in this PR or do you want me to create another PR ?

Reply:
I agree with the 6 months period. It sounds reasonable to me.

Reply:
Good one @ajantha-bhat, and I'm all in for adding @jbonofre to the list 👍